### PR TITLE
Convert Makefile.PL to cpanfile

### PR DESCRIPTION
**Pull request by mwiencek**
The cpanfile works with both carton and cpanm and doesn't require all this Module::Install crap (we're not publishing to CPAN).

### CODE REVIEW

**Comment by gentlecat** [comment]
No changes to `INSTALL.md` needed?

**Comment by mwiencek** [comment]
No changes needed, `cpanm --installdeps --notest .` still works perfectly fine.

**Comment by mwiencek** [comment]
Well, now it does. :) It seems the 'feature' specifiers aren't installed by default anymore, so I moved some modules to the top where needed. The server starts fine after a clean run of `cpanm --installdeps --notest .` now.

File: cpanfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ requires 'Catalyst::Action::RenderView'               => '0.16';
 requires 'Catalyst::Plugin::Authentication'           => '0.10023';
 requires 'Catalyst::Authentication::Credential::HTTP' => '1.016';
 requires 'Catalyst::Plugin::Cache'                    => '0.12';
+requires 'Catalyst::Plugin::Cache::HTTP'              => '0.001000';
```
Comment (mwiencek) [inline cpanfile]: `USE_ETAGS` is enabled by default, so the server would break by default without this.

**Pull request merged**